lms/djangoapps/verify_student/services.py

```diff
@@ -244,7 +245,8 @@ def get_verify_location(cls, course_id=None):
         location = f'{settings.ACCOUNT_MICROFRONTEND_URL}/id-verification'
         if course_id:
             location += f'?course_id={quote(str(course_id))}'
-        return location
+
+        return IDVPageURLRequested.run_filter(location)
```

Comment:
Don't forget to add:             
```
# .. filter_implemented_name: IDVPageURLRequested
# .. filter_type: org.openedx.learning.idv.page.url.requested.v1
```

So it's easier to identify later on. Thanks!